### ISSUE TITLE
User never goes idle in cordova

### ISSUE THREAD

**Issue by riaan53** [mizzao/meteor-user-status#47]
Hi,

Im using this package with meteor (v1.0.1) Cordova on iOS. It works 100% on browser and on mobile browser - but when in Cordova the user never goes idle. Any ideas? 

Thanks.

Regards,
Riaan

**Comment by mizzao** [comment]
Did you ever find anything on this? One way to test may be to fork the package and do some logging on the client side, just to see if the inactivity event is actually being sent to the server.

**Comment by KristerV** [comment]
So I've tested this.

Currently user-status only detects online-offline activity (offline only happens with timeout).

To implement idle user-status would just need to use

```JavaScript
document.addEventListener("pause", function() {
  Session.set('cordovaStatus', 'paused');
  Meteor.disconnect();
});

document.addEventListener("resume", function() {
  Session.set('cordovaStatus', 'resumed');
  Meteor.reconnect();
});
```

I am unable to understand what's going on inside this package so can't PR. If however Mizzao would tell me what needs to be done I'll happily give my first go with coffescript and try implement this.

**Comment by KristerV** [comment]
tested and works. HOWEVER when user goes idle by "pause" then latestActivity gets "Invalid Date" because mizzao:timesync just isn't handling cordova somehow. Tracing this now.

**Comment by KristerV** [comment]
ah couldn't figure it out... for some reason HTTP.get() just doesn't get to the server (no error is returned either).

**Comment by KristerV** [comment]
Ok, got the timesync issue fixed, PR here https://github.com/mizzao/meteor-timesync/pull/26

**Comment by mizzao** [comment]
This should be fixed, please try out v0.6.5 in your apps.
